apps/gateway-ui/src/components/WithdrawCard.tsx

```diff
@@ -45,13 +45,29 @@ export const WithdrawCard = React.memo(function WithdrawCard({
   const [address, setAddress] = useState<string>('');
   const [amount, setAmount] = useState<number>(0);
 
+  const handleBitcoinUri = (address: string) => {
+    const uri = new URL(address);
+    const amt = uri.searchParams.get('amount');
+    if (amt) {
+      setAmount(parseFloat(amt));
+    }
```

Comment:
Amounts are in BTC, not sats
https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki#user-content-Transfer_amount